alerter/BinancePumpAndDumpAlerter.py

```diff
@@ -43,7 +42,13 @@ def __init__(
         self.telegram = telegram
         self.report_generator = report_generator
 
+        self.logger = logging.getLogger("pump-and-dump-alerter")

```

Comment:
Shifted this up for logger info on TDP_nearest_hour to be sent. Shouldn't be an issue.